geoportal/tests/functional/test_layers.py

```diff
@@ -33,11 +33,10 @@
 
 from unittest import TestCase
 
-from tests.functional import (  # noqa
-    teardown_common as teardown_module,
-    setup_common as setup_module,
-    create_dummy_request, cleanup_db, create_default_ogcserver,
-)
+from tests.functional import (cleanup_db, create_default_ogcserver,
+                              create_dummy_request)
+from tests.functional import setup_common as setup_module  # noqa
+from tests.functional import teardown_common as teardown_module  # noqa
```

Comment:
If you want the way imports are done, please apply it on the whole code in one PR.